src/web/mjs/connectors/futabanet.mjs

```diff
@@ -0,0 +1,45 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class futabanet extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'futabanet';
+        super.label = 'がうがうモンスター (Futabanet Monster)';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://futabanet.jp';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'head title');
+        let id = uri.pathname;
+        let title = data[0].text.split(' | ')[0].trim();
+        return new Manga(this, id, title);
+    }
+
+    /**
+     *
+     */
+    _getMangaList( callback ) {
```

Comment:
May use newer method `async _getMangas()`

Reply:
I don't know how to use that... If I just change `_getMangaList( callback )` to `async _getMangas()`
I get an error saying, `callback is not defined`